public/app/features/alerting/unified/components/rule-editor/QueryRows.tsx

```diff
@@ -136,6 +136,7 @@ export class QueryRows extends PureComponent<Props, State> {
         return {
           ...item,
           refId: query.refId,
+          queryType: item.model.queryType ?? '',
```

Comment:
would it make sense to simply "copy" the `model` in the root object? Something like:

```
return {
  ...item,
  ...item.model,
  refId: query.refId
```

Not sure if in the future another property apart from the `queryType` will be needed.